src/arrayMethodSort.js

```diff
@@ -3,9 +3,39 @@
 /**
  * Implement method Sort
  */
+
+function compareAsStrings(a, b) {
+  const strA = String(a);
+  const strB = String(b);
+
+  if (strA > strB) {
+    return 1;
+  } else if (strA === strB) {
+    return 0;
+  }
+
+  return -1;
+}
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareAsStrings) {
```

Comment:
good job 🔥 

also, you can apply by default such function and it will work the same
```suggestion
  [].__proto__.sort2 = function(compareFunction = (a, b) => String(a) > String(b) ) {
```